api/data/seattle_db.go

```diff
@@ -147,21 +233,36 @@ func (c *Client) SeattleSearchOfficerByName(firstName, lastName string) ([]*Seat
 	return seattleMarshalOfficerRows(rows)
 }
 
-// SeattleFuzzySearchByName invokes seattle_fuzzy_search_officer_by_name_p
+// SeattleFuzzySearchByName returns an list of officers by their full name using fuzzy matching.
+// It searches the full historical roster list but only returns the last availabe roster entry. Entries 
+// are sorted by date in descending order.
 func (c *Client) SeattleFuzzySearchByName(name string) ([]*SeattleOfficer, error) {
 	rows, err := c.pool.Query(context.Background(),
 		`
+			WITH o AS (
+				SELECT
+					*,
+					row_number() over (partition by o.badge order by o.date desc) seqnum
+				FROM seattle_officers o
+				WHERE LOWER(first_name || ' ' || last_name) % LOWER($1)
```

Comment:
I...have to be honest, I have no idea hos this line works lol

Reply:
Is this because I didn't do like `o.first_name || ' ' || o.last_name`?

Reply:
I mean the `LOWER(...) % LOWER($1)` bit

Reply:
Oh yeah that was ripped straight from Barry's code. I don't actually understand what it does but it works I guess haha.

Reply:
Ah found the explanation, looks like it's a postgres specific term:

https://www.postgresql.org/docs/current/pgtrgm.html

```
text % text → boolean

Returns true if its arguments have a similarity that is greater than the current similarity threshold set by pg_trgm.similarity_threshold.
```